zib/vocabulary/generated/CodeSystem-COD472VEKT-2.16.840.1.113883.2.4.3.11.22.472.xml

```diff
@@ -0,0 +1,26 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+   <id value="2.16.840.1.113883.2.4.3.11.22.472"/>
+   <language value="en-US"/>
+   <text>
+      <status value="empty"/>
+      <div xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">No narrative is provided for definitional resources. A human-readable rendering can be found in the implementation guide(s) where this resource is used.</div>
+   </text>
+   <extension url="http://hl7.org/fhir/StructureDefinition/resource-effectivePeriod">
+      <valuePeriod>
+         <start value="2024-09-20T14:30:12+00:00"/>
+      </valuePeriod>
+   </extension>
+   <url value="urn:oid:2.16.840.1.113883.2.4.3.11.22.472"/>
+   <identifier>
+      <use value="official"/>
+      <system value="urn:ietf:rfc:3986"/>
+      <value value="urn:oid:2.16.840.1.113883.2.4.3.11.22.472"/>
+   </identifier>
+   <version value="2024-09-20T14:30:12"/>
+   <name value="COD472VEKT"/>
+   <title value="COD472-VEKT"/>
+   <status value="unknown"/>
+   <publisher value="ART-DECOR"/>
+   <description value="COD472-VEKT"/>
+   <content value="not-present"/>
```

Comment:
Can you check why this is empyt? Gives warnings in QA.

Reply:
ValueSets updated, this file no longer exists.